lib/tmuxinator/project.rb

```diff
@@ -147,6 +147,12 @@ def tmux_command
       yaml["tmux_command"] || "tmux"
     end
 
+    def tmux_has_session?(name)
+      sessions = `#{tmux_command} ls`
+
+      !sessions.match("^#{name}:")
```

Comment:
I'm a bit confused as to why there's a `!` here.

The way I read it, if the session exists, `session.match("^#{name}:")` should return a match. `not`ing a match would return `false`, which is returned from the method. Shouldn't the method return `true` if the session exists?

If you were trying to make a boolean from the match object, you can do `!!`. For example:
```ruby
!! "foo" #=> true
```